packages/terra-toggler/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-toggler",
+  "main": "lib/Toggler.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Toggler component that transitions content in and out.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-toggler",
+    "Toggler",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "terra-base": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-animate-height": "^0.9.5",
```

Comment:
Can we use pure css to animate height or do we need this dependency?

It looks like terra-clinical is also using react-transition-group
https://github.com/cerner/terra-clinical/blob/d231d190abdd6adb14fa9a1b31718307943eff33/packages/terra-clinical-slide-group/package.json#L33